src/stream/src/executor/backfill/snapshot_backfill/executor.rs

```diff
@@ -125,12 +128,14 @@ impl<S: StateStore> SnapshotBackfillExecutor<S> {
         debug!(epoch = ?first_upstream_barrier.epoch, "get first upstream barrier");
         let first_recv_barrier = receive_next_barrier(&mut self.barrier_rx).await?;
```

Comment:
We should probably add a docstring for all these different epochs, once snapshot backfill implementation stabilizes